src/spec-lang/tc/typecheck.ts

```diff
@@ -779,6 +895,130 @@ function tcIdBuiltin(expr: SId): SType | undefined {
     return BuiltinSymbols.get(expr.name);
 }
 
+/**
+ * Given the type of some state variable `type`, a 'data-structure path' `path` find the path of the
+ * element of the data structre pointed to by the data-structure path.
+ *
+ * @param type
+ * @param path
```

Comment:
These line can be removed as do not provide anything useful.

Reply:
Fixed in dcf955c.